weedcoco/index/indexing.py

```diff
@@ -30,6 +30,7 @@ def __init__(
         self,
         weedcoco_path,
         thumbnail_dir,
+        dataset_name="",
```

Comment:
I'm not sure why we need this override when we've got metadata["name"] within the weedcoco